spread_tests/scripts/prepare_pebble_runtime.sh

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+set -e
+
+PEBBLE_RUNTIME_DIR=$(${SCRIPTS_DIR}/get_pebble_runtime_dir.sh)
+mkdir -p "${PEBBLE_RUNTIME_DIR}"
+rm -rf "${PEBBLE_RUNTIME_DIR}/*"
```

Comment:
Dangerous waters here as a variable issue means /* :D 
```suggestion
rm -rf "${PEBBLE_RUNTIME_DIR:?}/*"
```